drivers/sub8_videoray_m5_thruster/launch/thruster_driver.launch

```diff
@@ -1,4 +1,4 @@
 <launch>
-    <node pkg="sub8_videoray_m5_thruster" type="thruster_driver.py" name="thruster_driver"
-        args="--calibration_path=$(find sub8_videoray_m5_thruster)/config/calibration.json" />
+  <param name='thrusters/calibration_dir' type='string' value='$find subb8_videoray_m5_thruster)/config/calibration/' />
```

Comment:
Restructured, removed individual launches in sub8_videoray and sub8_thruster_mapper and only thrusters.launch in sub8_launch.